bn_mp_export.c

```diff
@@ -7,14 +7,23 @@
  * see http://gmplib.org/manual/Integer-Import-and-Export.html
  */
 mp_err mp_export(void *rop, size_t *countp, int order, size_t size,
```

Comment:
countp == written??

Reply:
Maybe rename countp to written? But then we are diverging from the gmp api. But the API is already not 100% similar, since GMP also does internal allocation for rop==NULL. But we should keep it simple as it is.

Reply:
@czurnieden I don't think we need both countp and written?

Reply:
> countp == written??

No, they are not equal, they count two different things.

Reply:
written=count*size? We don't need this. mp_export is already complicated enough.

Reply:
We should also use enums for the flags. This will make things more clear. Maybe add this to this PR?

Reply:
> We should also use enums for the flags

As you are now our semi-official "namer of functions and stuff": any suggestions?